### DISCUSSION TITLE
Secure Websocket connections?

### DISCUSSION THREAD

**Discussion by martykazmaier** [mkrueger/icy_term#42] [General]
Is anyone using this?  I've got a websocket connection at shsbbs.net:8080 that should be using TLS, but it doesn't connect securely.  I'd like to see an example of this working so I can get this functional.  This isn't a problem with icyterm, but icyterm is the only place I've seen this feature implemented in a client.  I'd like to implement it in a server.

**Comment by martykazmaier** [comment]
Ok, I'm forcing SSL TLS connections in websockify (my proxy server) and now neither websocket or secure websocket is working with shsbbs.net:8080.  Ftelnet through my proxy is working fine (https://shsbbs.net/bbs.html).

**Comment by mkrueger** [comment]
Enigma BBS can do that works there.
